docs/src/explanations/connection-operators.md

```diff
@@ -2,7 +2,7 @@
 
 layout: docs
 
-title:  "Deep Dive into Connection Operators"
+title:  "Deep Dive into (chisel3 and Chisel) Connection Operators"
```

Comment:
Why did you change this title? The article doesn't say anything about a Chisel operators